### PR TITLE
Docker composed basic template

### PR DESCRIPTION
**Pull request by almiracle**

### CODE REVIEW

File: python/basic/README.md [hunk 1]
```diff
@@ -1,24 +1,39 @@
-## Maana Python Template
```
Comment (squanchd) [inline python/basic/README.md]: Maybe leave this?

File: python/basic/README.md [hunk 2]
```diff
@@ -1,24 +1,39 @@
...
+
+The first time you run this it might take some time, any time after will be quicker.
+
+The base image contains all of the packages and pips that the project is dependent on as well as a template for a microservice. Since this changes rarely we don't want to rerun this frequently. Once it is run once it will only be rerun if the contents of docker_base change.
+
+The service image is based off of the docker_base image. anything in docker_base can be modified and tailored to the service by adding the same filename to the service. All of the files in the service directory are copied over the files in the base directory and then server is started. This way, many services can use the same base image and have different resolvers, models, etc.
+
+# Docker Compose Up
+
+Docker-compose up brings up the built images as services.
```
Comment (squanchd) [inline python/basic/README.md]: We need to add the AKS deployments steps here.

Comment (squanchd) [reply python/basic/README.md]: ### Prerequisites 
1. Install Azure's AKS CLI 
`az aks install-cli`
2. Get credentials for the AKS cluster
`az aks get-credentials --resource-group <RESOURCE_GROUP> --name <AKS_CLUSTER_NAME>`
3. Open the Kubernetes dashboard 
` az aks browse --resource-group <RESOURCE_GROUP> --name <AKS_CLUSTER_NAME>`
4. List deployed assets (Pods, deployments)
`kubectl get all`

### Kubernetes Deployment
1. Create a name space in Kubernetes
`kubectl create ns <NAMESPACE_FOR_YOUR_SERVICE>`
2. Apply docker-compose.yml file to the namespace
kubectl apply -n <NAMESPACE_FOR_YOUR_SERVICE> -f <PATH_TO_DOCKER_COMPOSE>

### Expose your application
`kubectl expose deployment <DEPLOYMENT> --type=LoadBalancer --name=<NAME_FOR_EXPOSED_SERVICE> --namespace=<NAMESPACE_FOR_YOUR_SERVICE>`

File: python/basic/docker_base/schema/mutation.gql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+type Mutation {
+  test: String
```
Comment (squanchd) [inline python/basic/docker_base/schema/mutation.gql]: imho, the default endpoint should be "info". wdyt?

File: python/basic/docker_base/schema/query.gql [hunk 4]
```diff
@@ -0,0 +1,3 @@
+type Query {
+  test: String
```
Comment (squanchd) [inline python/basic/docker_base/schema/query.gql]: Same (The default endpoint should be "info")

File: python/basic/docker_base/shared/graphiql/graphiql.html [hunk 5]
```diff
@@ -0,0 +1,166 @@
+<!--
```
Comment (squanchd) [inline python/basic/docker_base/shared/graphiql/graphiql.html]: We should probably use one of our react app examples for this.

Comment (almiracle) [reply python/basic/docker_base/shared/graphiql/graphiql.html]: graphiql is good for being able to test the service locally. I think we should have a project template that has a react example and this both working together to show how these things interact.

File: python/basic/docker_base/schema/mutation.gql (hunk 3 above)
Comment (almiracle) [reply python/basic/docker_base/schema/mutation.gql]: yes, good point, will update

**Comment by Akavall** [comment]
I pulled it and tested it with "toy-service". It looks to be working fine. I think we can merge it.

**Pull request merged**